From: Rico Tzschichholz <ricotz@ubuntu.com>
Date: Tue, 8 Mar 2016 18:21:14 +0100
Subject: main: Properly construct OptionEntry

Stop using an empty string where is getting treated as translateable.

Fixes https://launchpad.net/bugs/1554618.
Cherrypicked from upstream (c4df2371a32973dc173d1bde7426f722e0441608)
---
 src/ui/synapse-main.vala | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/ui/synapse-main.vala b/src/ui/synapse-main.vala
index a319dbb..ab33857 100644
--- a/src/ui/synapse-main.vala
+++ b/src/ui/synapse-main.vala
@@ -28,7 +28,7 @@ namespace Synapse
     {
       {
         "startup", 's', 0, OptionArg.NONE,
-        out is_startup, "Startup mode (hide the UI until activated).", ""
+        out is_startup, "Startup mode (hide the UI until activated).", null
       },
       {
         null
